src/dpp/guild.cpp

```diff
@@ -213,13 +213,20 @@ std::string guild_member::build_json(bool with_id) const {
 			j["communication_disabled_until"] = json::value_t::null;
 		}
 	}
-	if (!this->nickname.empty())
+	
+	if (!this->nickname.empty()) {
 		j["nick"] = this->nickname;
+	} else {
+		j["nick"] = find_user(user_id)->username;
```

Comment:
just set the nick to `json::value_t::null` here. no need to search for the user in cache. The method should edit the fields using the passed object.

Reply:
ya it still work cool :">